Mathlib/CategoryTheory/Limits/IndYoneda.lean

```diff
@@ -74,7 +74,6 @@ lemma colimitHomIsoLimitYoneda_hom_comp_π [HasLimitsOfShape Iᵒᵖ (Type u₂)
   change ((coyonedaOpColimitIsoLimitCoyoneda F).hom ≫ _).app A = _
   rw [coyonedaOpColimitIsoLimitCoyoneda_hom_comp_π]
 
-@[reassoc (attr := simp)]
```

Comment:
It would be better to replace `(coyoneda.map ... .).app A` by `(yoneda.obj A).map ...`. (And similarly in the RHS of the lemma above.) You may have to fix a few (easy?) breakages in a other files.